packages/client/src/app/components/site-sidenav/site-sidenav.component.html

```diff
@@ -8,32 +8,32 @@ <h3>{{ currentUser.username }}</h3>
         <h4>Welcome back!</h4>
       </div>
       <div class="close-sidenav">
-        <a class="close-button" (click)="onCloseClicked()"><i-feather name="x"></i-feather></a>
+        <a matTooltip="Close" matTooltipClass="offprint-tooltip" class="close-button" (click)="onCloseClicked()"><i-feather name="x"></i-feather></a>
```

Comment:
Not something for this PR, but I was looking at this and thinking that there's plenty of space here for actual labels, not just icons + tooltips. Be an accessibility _and_ mobile win.